storagetransfer/storagetransfer_test.go

```diff
@@ -95,6 +110,29 @@ func TestMain(m *testing.M) {
 		log.Fatalf("couldn't create S3 bucket: %v", err)
 	}
 
+	queue := testutil.UniqueBucketName("stssqsqueue")
```

Comment:
TestMain is getting really long here. If this resource is not shared across tests can it go in the individual test function where it's used? Same for the pubsub topic below. I'd only keep them in here if they are resources to be shared across multiple tests.